datadog_cluster_agent/datadog_checks/datadog_cluster_agent/check.py

```diff
@@ -0,0 +1,47 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import OpenMetricsBaseCheck
+
+
+class DatadogClusterAgentCheck(OpenMetricsBaseCheck):
+    """
+    Collect Cluster Agent metrics from its Prometheus endpoint
+    """
+
+    def __init__(self, name, init_config, instances):
+        metrics = {
+            'admission_webhooks_certificate_expiry': 'admission_webhooks_certificate_expiry',
+            'admission_webhooks_mutation_attempts': 'admission_webhooks_mutation_attempts',
```

Comment:
metric naming usually follows a pseudo-hierarchical dotted format (e.g. http.nginx.response_time).

```suggestion
            'admission_webhooks_certificate_expiry': 'admission_webhooks.certificate_expiry',
            'admission_webhooks_mutation_attempts': 'admission_webhooks.mutation_attempts',
```